contrib/buildrpm/README

```diff
@@ -0,0 +1,74 @@
+
+# Copyright (c) 2016      Cisco Systems, Inc.  All rights reserved.
+#
+# Written by: Landon Curt Noll, Martin Kontsek
+#
+# Permission to use, copy, modify, and distribute this software and
+# its documentation for any purpose and without fee is hereby granted,
+# provided that the above copyright, this permission notice and text
+# this comment, and the disclaimer below appear in all of the following:
+#
+#       supporting documentation
+#       source copies
+#       source works derived from this source
+#       binaries derived from this source or from derived source
+#
+# Landon Curt Noll, Martin Kontsek DISCLAIMS ALL WARRANTIES WITH REGARD
+# TO THIS SOFTWARE, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+# AND FITNESS. IN NO EVENT SHALL Landon Curt Noll AND Martin Kontsek BE
+# LIABLE FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR ANY
+# DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
```

Comment:
@martinkontsek I'm not sure this license is compatible with the OpenFabrics mandate -- all submissions to libfabric must be GPL/BSD dual-licensed.  For example, see https://github.com/ofiwg/libfabric/blob/master/src/enosys.c#L2-L30.